csaf-validation/src/test/kotlin/io/github/csaf/sbom/validation/tests/TestsTest.kt

```diff
@@ -609,4 +636,26 @@ class TestsTest {
             )
         }
     }
+
+    /**
+     * Short utility function to construct the path to the test file based on the test file ID for
+     * mandatory tests.
+     */
+    fun mandatoryTest(id: String): String {
+        var test = "mandatory/oasis_csaf_tc-csaf_2_0-2021-${id}.json"
```

Comment:
This code wouldn't work for CSAF 2.1 test files. Is that hard coding intended? Why don't we extract that from the testdata.json?

Reply:
We previously did not use the testcases.json so this is (still) hard-coded. I intend to re-write this in a future PR so that we actually extract all the paths from the testcases.json, making this also more CSAF 2.1 proof. At least the "valid" cases we could just iterate from the testcases. For the failures ones, I still at least need to specify the ID -> future PR.